react/ThumbnailList.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import ThumbnailItem from './ThumbnailItem'
+
+class ThumbnailList extends Component {
+  render() {
+    const { images, onThumbnailClick } = this.props
+
+    return (
+      <div className="absolute dtc dt w-10">
```

Comment:
The classes `dtc` and `dt` both update the display, consider using only one. Probably `dt` since is the one that is active right now.